### PR TITLE
Update lint-test.yml

### PR DESCRIPTION
**Pull request by samsiegart**
We had this disabled

### CODE REVIEW

File: .github/workflows/lint-test.yml [hunk 1]
```diff
@@ -7,9 +7,6 @@ on:
   push:
     branches: [main]
   pull_request:
-  repository_dispatch:
-    types: [test-agoric-sdk]
-  # FIXME: Deprecated build from schedule.
   schedule:
     - cron: "0 12 * * *"
```
Comment (turadg) [inline .github/workflows/lint-test.yml:11]: does this run a nightly build/test even if nothing has changed?

**Review by turadg — APPROVED** [review]
🙌

File: .github/workflows/lint-test.yml (hunk 1 above)
Comment (samsiegart) [reply .github/workflows/lint-test.yml:11]: I think so, according to https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07. Maybe we shouldn't?

Comment (samsiegart) [reply .github/workflows/lint-test.yml:11]: I'll just merge it for now and we can take it out later if desired

**Pull request merged**